java_api/standard_api/src/main/java/uk/ramp/distribution/Distribution.java

```diff
@@ -87,6 +95,33 @@ private RealDistribution underlyingDistribution() {
       dist.load(
           empiricalSamples().orElseThrow().stream().mapToDouble(Number::doubleValue).toArray());
       return dist;
+    } else if (internalType().equals(DistributionType.categorical)) {
+      List<Integer> bins = bins().orElseThrow();
```

Comment:
I've lost where we were discussing this, and can't remember if I mentioned this to you already, but I did wonder whether the categories should be strings, and then parsed on the client side only when deemed appropriate?

Reply:
I've added a comment [here](https://github.com/ScottishCovidResponse/SCRCIssueTracking/issues/671) where we were discussing this. I agree with you. It will require quite a bit of work to support the string range format, i'll add that as a TODO, but let's keep this format for now as a first step. Would that be ok?

Reply:
Ah yeah, that's where it was! I mean yes, if it is a lot of work then that's fine - would adding in support for plain strings also be time consuming? I mean so that the format is this:
```
[population-ages]
type = "distribution"
distribution = "categorical"
bins = ["0", "15", "25", "55", "65", "90"]
weights = [0.1759, 0.1171, 0.4029, 0.1222, 0.1819]
```
Then we can think about the ranges later. If this is more than trivial then feel free to merge.

Reply:
Up to you - I'm just a bit worried about (effective) API lock-in, but I don't really think it's a big deal, this isn't something that is being broadcast to the world.

Reply:
@mrow84 updated PR to include implementation for your suggestion. Please could you review again. Thanks!